dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/blas/function/AbsoluteTest.java

```diff
@@ -44,10 +44,19 @@ public void setup() throws InjectionException {
         .build();
     final MatrixFactory matrixFactory = Tang.Factory.getTang().newInjector(conf).getInstance(MatrixFactory.class);
 
-    this.input = matrixFactory.create(new float[][]{{1.0f, -2.0f, 3.0f}, {-4.0f, 5.0f, -6.0f}});
-    this.expectedOutput = matrixFactory.create(new float[][]{{1.0f, 2.0f, 3.0f}, {4.0f, 5.0f, 6.0f}});
-    this.expectedDerivative = matrixFactory.create(new float[][]{
-        {1.0f, -1.0f, 1.0f}, {-1.0f, 1.0f, -1.0f}});
+    final int numInput = 3;
+    final int numBatch = 2;
+    this.input = matrixFactory.create(new float[]{
+        1.0f, -2.0f, 3.0f,
+        -4.0f, 5.0f, -6.0f},
+        numInput, numBatch);
```

Comment:
What's the reason for using `create(float[], int, int)` and not `create(float[][])` like before? There seems to similar changes in other test classes.